src/components/RootFolder/RootLayout.js

```diff
@@ -1,5 +1,14 @@
+/* eslint-disable import/no-extraneous-dependencies */
+
+// import { useContext } from 'react';
 import { NavLink, Outlet, useLocation } from 'react-router-dom';
 import './RootLayout.css';
+// import 'src/components/Page-1/Page1.module.css';
+// import 'src/components/Quote/Quote.module.css';
+// import 'src/components/Calculator-Folder/Calculator.module.css';
+// import SharedStateContext from '../SharedStateContext';
+// import Navigation from './Navigation';
+// import Footer from './Footer';
```

Comment:
- [ ] Kindly remove the commented-out code to keep the code base clean.

- [ ] Kindly don't disable the linter checks. Linter checks are in place in order to better structure your code and improve your code quality.